app/views/report/_form_columns.html.haml

```diff
@@ -26,6 +26,12 @@
       = render :partial => "column_lists"
     %strong
       = _('* Caution: Changing these fields will clear all selected values below them!')
+    - if @unavailable_fields
+      %br
+      %p{:style => "max-width: 850px;"}
+        %strong
+          = _("* Caution: %{unav_fields} are not supported for %{rep_base}.") % {:unav_fields => @unavailable_fields, :rep_base => Dictionary.gettext(@edit[:new][:model], :type => :model, :notfound => :titleize, :plural => true)}
```

Comment:
Using string arithmetics for gettext strings is generally a bad idea. It makes the work of translators more difficult, plus the resulting translated string will not always make sense.

Reply:
Hmm, that is interesting because I was told to do so in some other PR. So what is your suggestion instead of this? What is the best? Thank you.

Reply:
Well, you need to put the static parts of the string together in one place (in this case, probably the controller) and substitute `%{rep_base}` here (that's the only dynamic part, correct?)